build script for Crontab and spdystream packages

Sapana-Khemkar opened the pull request.

amitsadaphule reviewed (changes requested):
The PR does not seem to be constrained to the 2 packages. Shows 88 files changed. Also, there are conflicts. Please address these issues and then we can proceed with further review.

Sapana-Khemkar commented:
> The PR does not seem to be constrained to the 2 packages. Shows 88 files changed. Also, there are conflicts. Please address these issues and then we can proceed with further review.
Fixed conflict issue and now total files changed are 3.

amitsadaphule commented:
The PR still seems to include the build-script for an additional package `genproto`. Is that intentional?

Also, `spdystream` is missing LICENSE file.

Sapana-Khemkar commented:
> The PR still seems to include the build-script for an additional package `genproto`. Is that intentional?
> 
> Also, `spdystream` is missing LICENSE file.

My bad. By mistake I pushed new packages to this branch. reverted these changes. Also added missing LICENSE for spdystream

amitsadaphule commented on s/spdystream/spdystream-ubi8.4.sh:
```diff
@@ -0,0 +1,58 @@
+export GOPATH=/home/tester/go
+
+mkdir -p $GOPATH/$PACKAGE_NAME/src
+cd $GOPATH/$PACKAGE_NAME/src
```
Should this be

```
mkdir -p $GOPATH/src/github.com/moby
cd $GOPATH/src/github.com/moby
```

to be consistent with go conventions?

Amit-Sirohi replied:
Changes done

amitsadaphule commented on s/spdystream/spdystream-ubi8.4.sh:
```diff
@@ -0,0 +1,58 @@
+git checkout $PACKAGE_COMMIT_HASH
+
+#install dependencies
+go mod init github.com/spdystream
+go mod tidy
+
+go get ./...
```
The repo already contains go.mod. So, is it really necessary to call `go mod xx` and `go get xx` commands? Can't we directly do a `go build ./...` instead?

Sapana-Khemkar commented:
@seth-priya @kandarpamalipeddi can one of you review this PR. All review comments are already in place. I think Amit Sadaphule did not get chance to review n merge
